packages/ui/VirtualList/VirtualList.module.less

```diff
@@ -13,18 +13,32 @@
 	&.native {
 		overflow-y: scroll;
 	}
+
 	.listItem {
 		width: 100%;
 	}
+
+	&.padding {
+		.listItem {
+			width: calc(100% - 96px);
```

Comment:
ui lib is not a themed lib, so we need to avoid a specific design. Let's find another way to add paddings.